src/components/pagination-links.jsx

```diff
@@ -19,23 +19,22 @@ const getNextRoute = (router, offset) => ({
   }
 })
 
-const routingCallback = ({offset, routename, router, routingActions}, offsetSelector) => _ => routingActions(routename)(getNextRoute(router, offsetSelector(offset)))
+const routingCallback = (props, offsetSelector) => _ => props.routingActions(props.routename)(getNextRoute(props, offsetSelector(props.offset)))
 
-export default props => (
+export default props => {
+  return (
   <ul className="pager p-pagination-controls">
     {props.offset > 0 ?
       <li>
-        <Link onClick={routingCallback(props, minOffset)}
-              to={props.router.location.pathname}
-              query={offsetObject(minOffset(props.offset))}
+        <Link to={{pathname:'', query: offsetObject(minOffset(props.offset))}}
+              onClick={routingCallback(props, minOffset)}
```

Comment:
Because this usage is deprecated by react-router 2.0. Query params moved into `query` key in `to` prop